app/services/preserved_object_handler.rb

```diff
@@ -183,4 +183,8 @@ def string_to_int(val)
     return val.to_i if val.instance_of?(String) && val.scan(/\D/).empty?
     val
   end
+
+  def unexpected_version_status
+    @unexpected_version_status ||= Status.find_by(status_text: Settings.statuses.detect { |s| s == 'expected_version_not_found_on_disk' })
```

Comment:
i'd be inclined to do a `find_by!` here so it fails fast, but it'd fail soon enough w/ a `null` status being set if nothing gets found here.